scripts/docker-run.sh

```diff
@@ -38,6 +38,27 @@ else
 	DOCKER_RUN="docker run"
 fi
 
+#
+# Ensure that either UPSTREAM_PRODUCT_BRANCH or DELPHIX_PACKAGE_MIRROR_MAIN
```

Comment:
This comment refers to DELPHIX_PACKAGE_MIRROR_MAIN but I don't see that variable used in the logic below.